ArchaicQuestII.GameLogic/Core/CoreHandler.cs

```diff
@@ -41,6 +41,14 @@ public void Init(IApplicationBuilder app)
         Db = app.ApplicationServices.GetService<IDataBase>();
         Pdb = app.ApplicationServices.GetService<IPlayerDataBase>();
         
+        //Init Handlers (Dont Include Databases in init)
+        Character.Init();
+        Client.Init();
+        Combat.Init();
+        Command.Init();
+        Item.Init();
+        World.Init();
```

Comment:
Hi what do the init handlers do? Item and world are empty, I haven't checked the others